syntax = "proto3";

package small.catalog;

import "src/schema/schema.proto";

service Catalog {
  rpc CreateTable(CreateTableRequest) returns (Reply) {}
  rpc UpdateTable(small.schema.Table) returns (Reply) {}
}

message CreateTableRequest {
  string table_name = 1;
  repeated small.schema.Column columns = 2;
}

message Reply {
  bool success = 1;
}
